README.md

```diff
@@ -28,6 +29,9 @@ It is available for install from the [JetBrains Marketplace](https://plugins.jet
  - Open the Dashboard for the connected cluster (works on any Hybrid cloud cluster)
  - Support [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) for logs and terminals.
  - Scale the replicated pods for deployments, deployment configs, replica/stateful sets or replication controllers
```

Comment:
Pretty sure "deployment configs" were deprecated from the openshift api in 4.14, so now we just have deployments as far as I understand.

Reply:
@btjd thx for pointing out. Still, we still support these for backward compatibility if the user is running older OpenShift versions.